### ISSUE TITLE
Breaks some Apothecary ingredients, overwrites magnitude for effect #0 and #1 with 0 

### ISSUE THREAD

**Issue by spacepill0w** [EzioTheDeadPoet/Weightless-Misc-Items#5]
Hey, not sure what the best way to document this is, but this patcher seems to be overwriting some apothecary ingredients' magnitude, causing potions to say something like "Heals for 0 points for 15 seconds" or something along the lines of that. Seems to be affecting effect #0 and effect #1. Odd thing is that this only happening on *some* ingredients. Not all. Daedra heart, giant's toe, wheat, hanging moss.... maybe less than a 1/4th of ingredients are being affected. I honestly can't find a pattern in it, which is super odd. This can be fixed by manually overriding the esp with apothecary's changes, but is still a pain. Tell me if you need more information and I'd be happy to supply it.

Methodology: Synthesis.esp was overwriting Apothecary and skyrim's ingredient effects. I made testing.esp (a group that only had weightless-misc-items) and ran xEdit again. This patcher is the culprit.
![image](https://user-images.githubusercontent.com/68788165/229310373-341b166e-df22-4085-b5b9-302e03ab8164.png)

**Comment by spacepill0w** [comment]
Oh, there's a setting on by default that stops weightless items from healing. That's pretty awful seeing as it just breaks any ingredients that heal.

**Comment by EzioTheDeadPoet** [comment]
> Oh, there's a setting on by default that stops weightless items from healing. That's pretty awful seeing as it just breaks any ingredients that heal.

I noticed this in my main patcher this was derived from. And fixed it there. But I don't use this patcher much so it barely even is on my radar. 

I have set it to hidden in Synthesis by default now, since I am unsure when or if I will change this. 
But my goal was to put some balancing factor on health ingredients. Sadly my initial lack of knowledge how potion crafting works caused this error.

Considering that the behavior can be disabled though I guess you still have a patcher that works for you.
